### PR TITLE
LPS-161009 Fix Uniform suite

### PR DESCRIPTION
**Pull request by lucasperj**
https://issues.liferay.com/browse/LPS-161009

### CODE REVIEW

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:relevant

**Comment by lucasperj** [comment]
ci:test:relevant

**Comment by lucasperj** [comment]
ci:stop

**Comment by susanchen3** [comment]
ci:test:relevant

File: test.properties [hunk 1]
```diff
@@ -7449,6 +7451,7 @@
 
     test.batch.names[upgrade]=\
         database-upgrade-client-jdk8,\
+        functional-tomcat90-mysql57-jdk8
```
Comment (vicnate5) [inline test.properties:7454]: Typo here, will break the properties file

**Comment by vicnate5** [comment]
ci:stop

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:relevant

**Comment by lucasperj** [comment]
ci:stop

**Comment by lucasperj** [comment]
ci:stop

**Comment by lucasperj** [comment]
ci:stop

**Comment by vicnate5** [comment]
@lucasperj I do not see the clear cause of the issue here since CI isn't reporting the error well, and unfortunately the tool we have to check how CI will parse a job is not working right now.

But there is one major issue in the PQL that you should solve first before we try running this again (and this could be the cause)

Your PQL for the `*-upgrade-*` batches will also return non-upgrade tests. Remember to test all your PQL locally.
1. Set these in your test.USERNAME.properties
```
test.batch.property.query=\
      (\
            (portal.acceptance != quarantine) AND \
            (portal.upstream != quarantine)\
        ) AND \
        (\
            (testray.main.component.name == "Database Partitioning") OR \
            (testray.main.component.name == "Database Upgrade Framework") OR \
            (testray.main.component.name == "Database Upgrade Tool")\
        ) AND \
        (\
            (app.server.types == null) OR \
            (app.server.types ~ tomcat)\
        ) AND \
        (\
            (database.types == null) OR \
            (database.types ~ mysql)\
        )
test.batch.max.group.size=1
test.batch.run.type=single
```
2. Execute `ant -f build-test.xml record-test-case-method-names -Dtest.run.environment=EE`

In this case, it will return tests like `DatabasePartitioning#CanDeleteCompany`, which should not be running in the upgrade batch. 

It looks like we add `(data.archive.type != null)` to most of our PQL for upgrade batches to make sure that it only runs upgrade tests, can you try this out?

FYI @susanchen3

**Comment by lucasperj** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
See https://github.com/vicnate5/liferay-portal/pull/2134

**Comment by vicnate5** [comment]
ci:close

**Pull request closed without merging**